main/test/flix/CompilerSuite.scala

```diff
@@ -5,6 +5,8 @@ import ca.uwaterloo.flix.util.{FlixSuite, Options}
 class CompilerSuite extends FlixSuite(incremental = true) {
   implicit val options: Options = Options.TestWithLibAll
 
+  mkTestDir("main/test/flix/")
```

Comment:
@magnus-madsen This is not working. The problem is, that in `main/test/flix/` there is `Test.Def.Generalization.flix` and `Test.Derives.Hash.flix` that were not included in this files before. Should they be deleted or refactored?

Reply:
See if you can make them work